Gems/PhysX/Code/Editor/KinematicDescriptionDialog.h

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#if !defined(Q_MOC_RUN)
+#include <QDialog>
+#include <AzFramework/Physics/Configuration/RigidBodyConfiguration.h>
+#endif
+
+class QTreeView;
+
+namespace Ui
+{
+    class KinematicDescriptionDialog;
+}
+
+namespace PhysX
+{
+    namespace Editor
+    {
+        class KinematicDescriptionDialog : public QDialog
```

Comment:
suggestion: It would be nice to add a brief API comment for this type explaining what it does and why it's needed to help future maintainers